boa/src/builtins/infinity/mod.rs

```diff
@@ -0,0 +1,33 @@
+//! This module implements the global `Infinity` property.
+//!
+//! The global `Infinity` is a property of the global object. In other words,
+//! it is a variable in global scope.
```

Comment:
A better doc would be:
```suggestion
//! The global property `Infinity` is a numeric value representing infinity.
```

Reply:
fixed it, thanks!